lib/compat/wordpress-6.1/class-wp-theme-json-data-gutenberg.php

```diff
@@ -47,6 +47,10 @@ public function update_with( $new_data ) {
 		return $this;
 	}
 
+	public function create_with( $new_data ) {
```

Comment:
Turns out we don't have a great way to _clear_ existing data if it's not overwrite it. This creates a new object from scratch. May need to revisit the ability to _clear_ data but this works for now.